tests/queryAttribute.js

```diff
@@ -177,12 +177,14 @@ describe('Attribute query', function () {
         ]);
     });
 
-    it('regexp flag (s)', function () {
-        const matches = esquery(literal, '[value=/\f.\r/s]');
-        assert.includeMembers(matches, [
-            literal.body[0].declarations[0].init
-        ]);
-    });
+    if ((/^(?:[89]|\d{2,})\./).test(process.version)) {
```

Comment:
Would this be a little more readable?

```suggestion
    if (parseInt(process.version) >= 8) {
```